ui/src/utils/index.ts

```diff
@@ -175,13 +116,3 @@ export function formatMessage(id: string, type: string, data: any) {
     data,
   });
 }
-
-/**
- * @description 检查 WebSocket 是否已激活。
- *
- * @param ws - WebSocket 实例。
- * @returns 如果 WebSocket 已激活则返回 true，否则返回 false。
- */
-export function wsIsActivated(ws: WebSocket | undefined) {
-  return ws ? !(ws.readyState === WebSocket.CLOSING || ws.readyState === WebSocket.CLOSED) : false;
-}
```

Comment:
There are no notable discrepancies or problems identified within this code snippet. The author has included clear comments that guide the usage of each API call appropriately. No further optimizations appear needed for these functions given their current structure.